custom/enikshay/management/commands/mark_successfull_99dots_records.py

```diff
@@ -0,0 +1,110 @@
+                    for attempt in repeat_record.attempts):
+                episode = accessor.get_case(repeat_record.payload_id)
+                if episode.get_case_property('dots_99_registered') != 'true':
+                    cases_to_update.add(episode)
+
+        with open('set_99dots_to_registered.csv', 'w') as f:
+            writer = csv.writer(f)
+            writer.writerow([
+                'beneficiary_id',
+                'episode_id',
+                'UpdatePatient Status',
+                'Adherence Status',
+                'TreatmentOutcome Status'
+            ])
+            print("Updating {} successful cases in 99DOTS".format(len(cases_to_update)))
+            for case in with_progress_bar(cases_to_update):
+                writer.writerow([
+                    get_person_case(domain, case.case_id).case_id,
+                    case.case_id,
+                    self.update_registered_status(domain, case),
+                    self.update_patients(domain, case),
+                    self.send_adherence(domain, case),
+                    self.send_treatment_outcome(domain, case),
+                ])
+
+    def update_registered_status(self, domain, case):
+        if self.commit:
+            update_case(domain, case.case_id, {'dots_99_registered': 'true'})
+
+    def update_patients(self, domain, case):
+        repeater_id = 'b4e19fd859f852871703e8e32a1764a9'
+        repeater_type = 'NinetyNineDotsUpdatePatientRepeater'
+        return self.send_repeat_record(domain, case.case_id, repeater_id, repeater_type)
+
+    def send_adherence(self, domain, case):
+        repeater_id = '9cf9a0e2df49b6271573afe677725dc2'
+        repeater_type = 'NinetyNineDotsAdherenceRepeater'
+
+        adherence_cases = get_adherence_cases_from_episode(domain, case.case_id)
+        if adherence_cases:
+            return {
+                adherence_case.case_id: self.send_repeat_record(domain, case.case_id, repeater_id, repeater_type).state
+                for adherence_case in adherence_cases
+            }
+        else:
+            return "No Adherence Cases"
+
+    def send_treatment_outcome(self, domain, case):
+        repeater_id = '3d912c5d00c73e9de5eb6a11c52a7301'
+        repeater_type = 'NinetyNineDotsTreatmentOutcomeRepeater'
+        if case.get_case_property('treatment_outcome', '') != '':
+            return self.send_repeat_record(domain, case.case_id, repeater_id, repeater_type).state
+        else:
+            return "No Treatment Outcome"
+
+    def send_repeat_record(self, domain, case_id, repeater_id, repeater_type):
+        repeat_record = RepeatRecord(
+            repeater_id=repeater_id,
+            repeater_type=repeater_type,
+            domain=domain,
+            next_check=datetime.utcnow(),
+            payload_id=case_id
+        )
+
+        if self.commit:
+            repeat_record.fire()
+
+        return repeat_record
```

Comment:
Might want to consider returning status only since thats the only things thats needed from repeat_record anyway.